scripts/dual_side_teleop.py

```diff
@@ -154,4 +164,10 @@ def main() -> None:
 
 
 if __name__ == '__main__':
-    main()
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--gravity_compensation',
+        action='store_true',
+        help='If set, gravity compensation will be enabled for the leader robots.',
```

Comment:
If set, gravity compensation will be enabled for the leader robots when teleop starts.